apps/api/trade-simulator-docker/Dockerfile

```diff
@@ -20,6 +20,7 @@ COPY . .
 RUN pnpm --filter=@recallnet/db build && \
```

Comment:
do we even need to filter these out anymore? we've gotten rid of the legacy blockchain deps as well as the problematic/autogen'd `api-sdk` & `api-mcp`